flight/PiOS/posix/pios_sys.c

```diff
@@ -54,26 +54,86 @@
 
 #include "pios_fileout_priv.h"
 #include "pios_com_priv.h"
+#include "pios_spi_posix_priv.h"
+#include "pios_ms5611_priv.h"
```

Comment:
Command line processing (and a lot of stuff that would be usually done in pios_board) happens here, so the same includes that pios_board.c would need to init the hardware are required.